crowdloans/rewards/polkadot.json

```diff
@@ -1227,7 +1239,8 @@
     },
     "parallelBonus": {
       "items": [
-        { "formula": "0.05", "logo": "/static/media/para.e1be6567.svg" }
+        { "formula": "0.30", "logo": "/static/media/para.e1be6567.svg" },
+        { "formula": "0.25" }
```

Comment:
Is this for Litentry? 
para reward is 30% and lit reward is 25%.

Reply:
yes